modules/custom/az_news/config/install/core.entity_view_display.node.az_news.az_card.yml

```diff
@@ -221,6 +229,7 @@ hidden:
   field_az_link: true
   field_az_main_content: true
   field_az_media_image: true
+  field_az_metatag: true
```

Comment:
```suggestion
```
References to metatag should probably be removed; metatag is part of `az_seo` and is optional.